build_runner/lib/build_runner.dart

```diff
@@ -4,6 +4,7 @@
 export 'src/asset/file_based.dart';
 export 'src/asset/reader.dart' show RunnerAssetReader;
 export 'src/asset/writer.dart';
+export 'src/build_script_generate/build_script_generate.dart';
```

Comment:
can you add a `show` with what we need explicitly? I'm worried we'll add some other public function that we don't want to leak.

Reply:
Done.